aws/rakefiles/deploy.rake

```diff
@@ -165,16 +165,25 @@ task :install_charts => [:configure_kubectl, :generate_modules, :setup_system_co
     else
       chart_namespace = 'default'
     end
+    if attrs && !attrs['allow_upgrade'].nil? && attrs['allow_upgrade'] == false
+      allow_upgrade = false
+    else
+      allow_upgrade = true
+    end
```

Comment:
Perhaps this if block can be replaced by:
```
allow_upgrade = attrs.fetch('allow_upgrade', true)
```
or something similar in order to see clearly that by default the value is **true**.

Reply:
Nice suggestion!